test/specs/accounts/update.account.spec.js

```diff
@@ -1,57 +1,40 @@
 'use strict';
-let config = require('../../../config.json');
 let loginPage = require('../../pages/login.page');
 let topSideBar = require('../../pages/topsidebar.page');
-let content = require('../../pages/accounts/acountContent.page');
-let form = require('../../pages/accounts/accountForm.page');
-let expect = require('chai').expect;
+let content = require('../../pages/content.page');
+let form = require(`../../pages/accounts/accountForm.${theme}.page`);
+
 describe('Acceptance Test to Modify an Account', function () {
-    this.retries(3);
 
-    let accountModifiedData = {
-        name: 'Account Updated',
+    let accountModified = {
+        name: 'Account Sample',
         number: 100,
         site: 2,
-        parent: 'Edge Communications',
```

Comment:
Maybe you need use this to test AutoCompleter Method.

Reply:
To run in lightning and classic I decided to remove some, but I will add again for classic